### PR TITLE
'Previous Cohorts are now hard to find' changes

### PR DESCRIPTION
**Pull request by 1Joda**
Created 'cohorts.md' to be the cohorts page

### CODE REVIEW

**Review by NPDebs — COMMENTED** [review]
Looking really good!! 🙌 💯

File: cohorts.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+ </div>
+
+ 
+ <div class="flip-card">
+  <div class="flip-card-inner">
+    <div class="flip-card-front">
+    <h1>OLS - 3</h1>
+    </div>
+    <div class="flip-card-back">
+      <a href='ols-1'><h2>Click here to find out more about the fifth cohort!</h2></a>
```
[the remark this answers is not in the document]
Comment (1Joda) [reply cohorts.md]: Omg thanks. I have made the changes!

[the remark this answers is not in the document]
Comment (NPDebs) [reply cohorts.md]: 😅 Sure. You're welcome.

File: _includes/default-header.html [hunk 2]
```diff
@@ -42,6 +41,7 @@
               <a class="navbar-item" href="/ols-6#mentors"> Mentors & Experts </a>
               <a class="navbar-item" href="/ols-6#collaborators"> Collaborators </a>
               <a class="navbar-item" href="/ols-6#resources"> Resources </a>
+              <a class="navbar-item" href="/cohorts"> Previous Cohorts </a>
```
Comment (bebatut) [inline _includes/default-header.html:44]: I do not think inside OLS-6 dropdown menu is the best location. I would prefer to leave in the OLS program drop-down menu

File: cohorts.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
+---
+layout: page
+title: All Open Life Science Cohorts till Date!
```
Comment (bebatut) [inline cohorts.md:3]:
```suggestion
title: OLS Cohorts
```

File: cohorts.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+    <h1>OLS - 6</h1>
+    </div>
+    <div class="flip-card-back">
+      <a href='ols-6'><h2>Click here to find out more about the sixth cohort!</h2></a>
+    </div>
+  </div>
+ </div>
+</div>
+
+# The OLS Program
```
Comment (bebatut) [inline cohorts.md:83]: I do not think that should go the cohort page

File: cohorts.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+# Cohorts
+
+<div class='cohort-grid'>
+  <div class="flip-card">
+  <div class="flip-card-inner">
+    <div class="flip-card-front">
+    <h1>OLS - 1</h1>
+    </div>
+    <div class="flip-card-back">
+   <a href='ols-1'><h2>Click here to find out more about the first cohort!</h2></a>
```
Comment (bebatut) [inline cohorts.md:20]: Which information does it show there?

File: cohorts.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+  url: https://flic.kr/p/5BXB6s
+---
+
+# Cohorts
+
+<div class='cohort-grid'>
+  <div class="flip-card">
+  <div class="flip-card-inner">
+    <div class="flip-card-front">
+    <h1>OLS - 1</h1>
```
Comment (bebatut) [inline cohorts.md:17]:
```suggestion
    <h1>OLS-1</h1>
```

Comment (bebatut) [reply cohorts.md:17]: It would be maybe nice to have the date or something else here visible

File: cohorts.md [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+image: /images/syllabus.jpg
+photos:
+  name: Niklas Morberg
+  license: CC BY-NC 2.0
+  url: https://flic.kr/p/5BXB6s
+---
+
+# Cohorts
+
+<div class='cohort-grid'>
```
Comment (bebatut) [inline cohorts.md:13]: To create the cards, could you use a loop over the cohorts? It would avoid having to have to add manually always new cohort

File: ols-1.md [hunk 8]
```diff
@@ -43,55 +43,15 @@ photos:
 {% endfor %}
 {% assign p-speakers = all-speakers | remove_first: ', ' | split: ", " | uniq | sort %}
 
-# The OLS-1 program
```
Comment (bebatut) [inline ols-1.md:46]: Why did you remove this content here? I think it should stay here. Same for the other cohort files

Comment (1Joda) [reply ols-1.md:46]: I moved it to the general cohort page but i see you want me to return it so i will.

**Comment by 1Joda** [comment]
I have noted your comments and will begin editting.
Thank you!!

File: cohorts.md (hunk 5 above)
Comment (1Joda) [reply cohorts.md:20]: It just takes you to the cohorts page

Comment (1Joda) [reply cohorts.md:20]: Do you want me to include things to the card like the year, timeline (i mean from 20xx-20xx) or any other thing?

File: cohorts.md (hunk 7 above)
Comment (1Joda) [reply cohorts.md:13]: I have done this now. Please let me know if its ok and if there's any other thing you'd like me to add to the card so it doesn't look so bare
Thank you!

File: cohorts.md (hunk 5 above)
Comment (bebatut) [reply cohorts.md]: Yes it would be good I think

File: ols-1.md [hunk 9]
```diff
@@ -52,15 +53,15 @@ Open Science skills
 **Outcome**: Ambassadors for Open Science practice, training and education across multiple European
 and international bioinformatics communities.
 
-**Process**: A 15-week mentoring & training program, based on the [Mozilla Open Leader program](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/), helping participants in becoming Open Science ambassadors by using three principles:
+**Process**: A 15 week mentoring & training program, based on the [Mozilla Open Leader program](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/), helping participants in becoming Open Science ambassadors by using three principles:
```
Comment (AndreaSanchezTapia) [inline ols-1.md:56]: please keep the hyphen here and in other places

File: _cohorts/ols-1.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (bebatut) [inline _cohorts/ols-1.md:1]: Why did you create this file and for other cohorts? Such files already exist

File: cohorts.md [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+          <a href="/{{ cohort.link }}"><h2>Click here to find out more about the {{ cohort.number }} cohort!</h2></a>
+        </div>
+      </div>
+    </div>
+ {% endfor %}
+</div>
+
+# OLS Cohorts - An Overview
+
+<!-- OLS-1 -->
```
Comment (bebatut) [inline cohorts.md:30]: This part could move at the top of the page and then you could use the information extracted there in the card (like dates, etc)

File: ols-1.md [hunk 12]
```diff
@@ -43,7 +43,8 @@ photos:
 {% endfor %}
 {% assign p-speakers = all-speakers | remove_first: ', ' | split: ", " | uniq | sort %}
 
-# The OLS-1 program
+# The OLS-1 Program
+
```
Comment (bebatut) [inline ols-1.md:47]: Please remove the extra line here

File: ols-1.md [hunk 13]
```diff
@@ -52,15 +53,15 @@ Open Science skills
...
+**Process**: A 15 week mentoring & training program, based on the [Mozilla Open Leader program](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/), helping participants in becoming Open Science ambassadors by using three principles:
 
 1. **Sharing** essential knowledge required to create, lead, and sustain an Open Science project.
 2. **Connecting** members across different communities, backgrounds, and identities by creating space in this program for them to share their experiences and expertise.
 3. **Empowering** them to become effective Open Science ambassadors in their communities.
 
 {% include _includes/toc.html %}
 
-# Goals and Learning Objectives
+# Our Goals and Learning Objectives
```
Comment (bebatut) [inline ols-1.md:64]: Could you leave the titles as they were? Thanks

File: cohorts.md (hunk 11 above)
Comment (1Joda) [reply cohorts.md:30]: I dont understand
I should move the overview up and the cards down but i should still include the things in the overview on the cards? should i just pick some or all?

File: ols-1.md (hunk 9 above)
Comment (1Joda) [reply ols-1.md:56]: will do!

File: _cohorts/ols-1.md (hunk 10 above)
Comment (1Joda) [reply _cohorts/ols-1.md:1]: I cant find any that has or that i can edit to include what i need
can you point me in the right direction?

File: cohorts.md (hunk 11 above)
Comment (1Joda) [reply cohorts.md:30]: Or do you just want me to move "OLS COHORTS - AN OVERVIEW" to the top of the page and not the whole table

Comment (1Joda) [reply cohorts.md:30]: Screenshot (47)
With this picture, am i on track with what you want me to do? Any way it would fit better with what you have in mind?
If i do include all the information from the table into the cards, will the table still be needed?

File: ols-1.md [hunk 14]
```diff
@@ -43,7 +43,7 @@ photos:
 {% endfor %}
 {% assign p-speakers = all-speakers | remove_first: ', ' | split: ", " | uniq | sort %}
 
-# The OLS-1 program
+# The OLS-1 Program
```
Comment (bebatut) [inline ols-1.md:46]: Can you revert your changes in this file and other `ols-x.md`?

Comment (1Joda) [reply ols-1.md:46]: im not sure how to go about that

File: _cohorts/ols-1.md (hunk 10 above)
Comment (1Joda) [reply _cohorts/ols-1.md:1]: It doesn't include any of the variables i used. Should i add them?

And you suggested I reuse the code for the table in the card but I don't think that is possible because each cohort was assigned its own variable.

Comment (bebatut) [reply _cohorts/ols-1.md:1]: Which variables do you need that is not in the Markdown file or in the metadata files in _data folder?

Comment (1Joda) [reply _cohorts/ols-1.md:1]: the name of the cohort {'OLS-1, OLS-2, etc} and the link to the respective .md file

If you see in the table that's currently in the .md file; each cohorts name and link are written down separately and not given as variables

Comment (bebatut) [reply _cohorts/ols-1.md:1]: True. With the current structure of the data, it is complicated to loop on the different cohorts. For now, you can remove the loop there, do like in the table for each cohort and create the cards manually

Comment (1Joda) [reply _cohorts/ols-1.md:1]: ok then! I'll try do that
and include the date at the front of the card

is there anything i should put at the back of the card, or should i leave it as is?

File: ols-1.md (hunk 14 above)
Comment (AndreaSanchezTapia) [reply ols-1.md]: You can make the changes (ex. put the hyphen and the old titles back) and commit these changes, @1Joda

**Comment by yochannah** [comment]
closing this as diverged from the current main branch
